### ISSUE TITLE
Can people outside of BU join the study group?

### ISSUE THREAD

**Issue by vnsriniv** [bulib/studyGroup#90]
Hi all,

I am a postdoc at Northeastern University working on microbial metagenomics and bioinformatics. I would really love to join this study group and was wondering if people outside of BU can join. 

Thanks
Varun

**Comment by wkearn** [comment]
Certainly! We haven't been particularly active this semester, but if you follow along on this repository or sign up on our newsletter at http://study.bu.edu you should hear about any thing we have going.

And if there is something you'd like to learn or share, feel free to open an issue, and we'll try to organize a session about it.

**Comment by vnsriniv** [comment]
Thanks! I registered for the session this week!

**Comment by tomhohenstein** [comment]
great! hope to see you at some upcoming events.
